antora-playbook.yaml

```diff
@@ -0,0 +1,17 @@
+site:
+  title: CMOS CI
```

Comment:
Should this be called something else? The value here is user-facing:
![image](https://user-images.githubusercontent.com/2904440/136784732-68566e44-e749-453f-8b7e-19dcdcefe284.png)